deepinv/models/poisson2sparse.py

```diff
@@ -0,0 +1,334 @@
+# Adapted from https://github.com/tacalvin/Poisson2Sparse
```

Comment:
include these attributions into the docstrings to be more visible + more precise